tests/src/tests/endpoint-test-utils.go

```diff
@@ -0,0 +1,59 @@
+package tests
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+var Exposures = [...]schema.EndpointExposure{schema.PublicEndpointExposure, schema.InternalEndpointExposure, schema.NoneEndpointExposure}
+
+func GetRandomExposure() schema.EndpointExposure {
+	return Exposures[GetRandomNumber(len(Exposures))-1]
+}
+
+var Protocols = [...]schema.EndpointProtocol{schema.HTTPEndpointProtocol, schema.WSEndpointProtocol, schema.TCPEndpointProtocol, schema.UDPEndpointProtocol}
```

Comment:
should `wss` and `https` be included as well?

Reply:
At the time of writing the tests the parser was not using the schema with this update. If it is now I can extend the test.

Reply:
yes, it's been merged in api repo as a part of https://github.com/devfile/api/pull/268